src/tests/helpers.js

```diff
@@ -919,6 +930,13 @@ export const schemaInputTypeSelectMultipleOptional = JSONSchemaBuilder()
   })
   .build();
 
+export const schemaInputTypeInteger = JSONSchemaBuilder()
```

Comment:
Some of the comments I mentioned at #10 are applied here too. I'll wait for you to fix those, re-applied them here, and then we can merge this one. It looks great!